### PR TITLE
WIP: DO NOT MERGE: Apache Arrow 2022-07 board report

### PR DESCRIPTION
**Pull request by kou**
This is a draft of Apache Arrow 2022-07 board report. Committers can
push directly to this branch. Others can leave suggestions.

This is never merged to apache/arrow. This will be reported by https://reporter.apache.org/wizard/?arrow .

### CODE REVIEW

File: board-report-2022-07.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- JULIA-2.3.0 was released on 2022-04-12.
+
+## Community Health:
+
+- A book about Apache Arrow was published from a community member.
+- We've split Apache Arrow Ballista to
+  https://github.com/apache/arrow-ballista from
+  https://github.com/apache/arrow-datafusion .
+- We've marked Apache Arrow C Streaming Interface stable.
+- A conference that focuses on Apache Arrow was held by community members.
```
Comment (paliwalashish) [inline board-report-2022-07.md:44]: Can we add the link to the conference (or the youtube channel)? Just a suggestion, it shall make it easy for the reader to explore further if interested. Same suggestion for book name

File: board-report-2022-07.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- RS-14.0.0 was released on 2022-05-16.
+- 8.0.0 was released on 2022-05-07.
+- RS-13.0.0 was released on 2022-05-03.
+- RS-12.0.0 was released on 2022-04-19.
+- RS-DF-7.1.0 was released on 2022-04-18.
+- JULIA-2.3.0 was released on 2022-04-12.
+
+## Community Health:
+
+- A book about Apache Arrow was published from a community member.
```
Comment (zeroshade) [inline board-report-2022-07.md:39]: Should a link to anything (article, Amazon, etc) for the book be included?

**Comment by kou** [comment]
What do you think about the following description in Board Reporting Guidelines for Project Chairs and detailed link for the conference and the book?

https://www.apache.org/foundation/board/reporting#donot

> Things not to put in reports
>
> *  Do not let commercial activity related to a project dominate a report.
>
>    Each Apache project's [PMC manages the project](https://www.apache.org/foundation/board/community.apache.org/projectIndependence) — not any third party or outside organization. Project reports should be about the project, and should not discuss outside organizationss activities unless they directly relate to the health of the project.

I'm not a native English speaker. So I can't determine that we should add a detailed link or something for the conference and the book...

File: board-report-2022-07.md (hunk 2 above)
Comment (pitrou) [reply board-report-2022-07.md:39]: Or at least the book title and author.

**Comment by nealrichardson** [comment]
By my reading of that policy, you could include the links--it wouldn't be "dominating" the report to do so. But that said, this report just goes to the board, they may reply with one line of comment, and that's it, right? I see they are published [here](https://whimsy.apache.org/board/minutes/Arrow.html) but I can't imagine who reads them. So maybe it doesn't matter.

**Comment by kou** [comment]
@nealrichardson Thanks! OK. I've added links for the book and the conference.

**Comment by kou** [comment]
Submitted: https://whimsy.apache.org/board/agenda/2022-07-20/Arrow

**Pull request closed without merging**